readme.md

```diff
@@ -638,22 +569,9 @@ Recipients
 	// if you need this...
 	myRecipient.Card = new StripeCreditCardOptions()
 	{
-		// set this property if using a token
-		TokenId = *tokenId*,
-
-		// set these properties if passing full card details (do not
-		// set these properties if you set TokenId)
-		Number = "4242424242424242",
```

Comment:
I wonder if it might be worth moving the credit card method to a subsection somewhere just so that if a user really wants to use it, they'll have somewhere that they can reference to see how it's done.

e.g. "A number of Stripe endpoints can take a credit card as an alternative to a token, but you should use this feature sparingly. An example of this usage might look like the following:

``` c#
myRecipient.Card = new StripeCreditCardOptions()
{
    Number = "4242424242424242",
    ...
}
```